apps/dash-manufacture-spc-dashboard/assets/spc-custom-styles.css

```diff
@@ -0,0 +1,794 @@
+/* Table of contents
+––––––––––––––––––––––––––––––––––––––––––––––––––
+- Banner
+- Modal
+- Tabs
+- Main Dashboard Tab
+- Measurement Tab
+- Tables/Dropdown
+- Containers
+- Media Queries
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+
+* {
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+body {
+  background-color: #1e2130;
+  color: #f3f5f4;
+  font-family: "Open Sans", sans-serif;
+  width: 100%;
+  height: 100vh;
+  max-width: 100% !important;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+#big-app-container {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 6rem;
+}
+
+.banner {
+  height: fit-content;
+  background-color: #1e2130;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: darkgray solid 1px;
+  padding: 1rem 4rem;
+  width: 100%;
+}
+
+
+.banner h5 {
+  font-family: 'Open Sans Semi Bold', sans-serif;
+  font-weight: 500;
+  line-height: 1.2;
+  font-size: 2rem;
+  letter-spacing: 0.1rem;
+  text-transform: uppercase;
+}
+
+.banner h6 {
+  color: darkgray;
+  font-size: 1.6rem;
+  line-height: 1;
+}
+
+.banner button {
+  color: #95969A;
+  margin-right: 10px;
+  height: 30px;
+  line-height: 30px;
+}
+
+#banner-logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.banner button:hover {
+  color: #92e0d3;
+  border-color: whitesmoke;
+}
+
+.banner Img {
+  height: 3rem;
+  margin: 0px 10px;
+}
+
+/* The Modal (background) */
+.modal {
+  display: none; /*Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1005; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100vw; /* Full width */
+  height: 100vh; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content/Box */
+.markdown-container {
+  width: 60vw;
+  margin: 10% auto;
+  padding: 10px 15px;
+  background-color: #2d3038;
+}
+
+.close-container {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  height: 40px;
+}
+
+.closeButton {
+  padding: 0 15px;
+  font-weight: normal;
+  float: right;
+  height: 100%;
+  border: none;
+}
+
+.closeButton:hover {
+  color: #91dfd2;
+}
+
+/* Tabs
```

Comment:
Series of things you can do to change the border around the tabs:

1. Change background to `background-color: #23262E;` under `.custom-tabs`
2. Change background color to `background-color: #161a28;` and border to `border: 0; border-bottom: #23262E solid 6px !important;` under `#Specs-tab.custom-tab, #Control-chart-tab.custom-tab`
4. Change border to `border-bottom: #91dfd2 solid 6px !important;` under `#Specs-tab.custom-tab--selected, #Control-chart-tab.custom-tab--selected`
5. Add the following portion of css:
```
#Specs-tab.custom-tab {
  margin-right: 3px;
}
#Control-chart-tab.custom-tab {
  margin-left: 3px;
}
```

It would then look as follows:
<img width="1680" alt="Screen Shot 2019-07-23 at 14 35 36" src="https://user-images.githubusercontent.com/23238764/61737906-27066000-ad57-11e9-91ab-6ccb73d63aee.png">

Reply:
💯  This margin could be set at the default breakpoint for dcc.Tabs when tab flex-direction changes from row to column
```
@media screen and (max-width: 800px) {
  #Specs-tab.custom-tab {
    margin-right: 0;
  }

  #Control-chart-tab.custom-tab {
    margin-left: 0;
  }
}
```